src/sys/socket/mod.rs

```diff
@@ -173,6 +170,47 @@ libc_bitflags!{
     }
 }
 
+/// Unix credentials of the sending process.
+///
+/// This struct is used with the `SCM_CREDENTIALS` ancillary message for UNIX sockets.
```

Comment:
This message, while true, is red herring because Nix doesn't support `SCM_CREDENTIALS`.  It should read "This struct is used with the `SO_PEERCRED`/`LOCAL_PEERCRED` socket options for UNIX sockets.